src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/CodeGeneration/CodeWriterExtensions.cs

```diff
@@ -804,13 +802,14 @@ private class LinePragmaWriter : IDisposable
                 _writer.WriteLine("#nullable restore");
             }
 
+            var sourceSpan = RemapFilePathIfNecessary(span, context);
```

Comment:
Can we add a comment explaining why we do not want the remapped file paths in other places (so file path comparisons are unaffected if I understand correctly)?

Reply:
There's a comment, and a link to the issue in the remap call itself. Do you still think we need another one here?